components/Starknet/modules/architecture_and_concepts/pages/Accounts/deploying_new_accounts.adoc

```diff
@@ -1,83 +1,48 @@
 [id="deploying_new_accounts"]
-= Deploying new accounts
+= Deploying a new account
 
-Starknet provides the `deploy_account` transaction in order to deploy new accounts to the
-network.
+You can deploy a new account in the following ways:
 
-== Deploy account transaction
+* Send a `DEPLOY_ACCOUNT` transaction. This method does not require a preexisting account.
+* Include the `deploy` syscall in the account contract, such as by invoking the Universal Deployer Contract (UDC) with an `INVOKE` transaction. This method requires an existing account to send the `INVOKE` transaction.
```

Comment:
```suggestion
* Using the Universal Deployer Contract (UDC). This method requires an existing account to send the `INVOKE` transaction.
```
Also suggest referencing the UDC documentation

Reply:
Fixed.